lib/virt_feature_test_base.pm

```diff
@@ -47,13 +47,70 @@ sub run_test {
     die('Please override this subroutine in children modules to run desired tests.');
 }
 
+sub prepare_run_test {
+    my $self = shift;
+
+    my $wait_script              = "600";
+    my $logs_cleanup_script_name = "clean_up_virt_logs.sh";
+    my $logs_cleanup_script_url  = data_url("virt_autotest/$logs_cleanup_script_name");
+
+    script_run("source /usr/share/qa/qa_test_virtualization/shared/standalone");
+    my $download_bash_script = "curl -s -o ~/$logs_cleanup_script_name $logs_cleanup_script_url";
+    script_output($download_bash_script, $wait_script, type_command => 0, proceed_on_failure => 0);
+    save_screenshot;
+    my $execute_bash_script = "chmod +x ~/$logs_cleanup_script_name && ~/$logs_cleanup_script_name -l \"/var/log/libvirt /var/lib/libvirt /var/log/xen\"";
+    script_output($execute_bash_script, $wait_script, type_command => 0, proceed_on_failure => 0);
+    save_screenshot;
+
+    $self->start_monitor_guest_console;
+}
+
+sub guest_console_monitor {
```

Comment:
Function name is better to start with a verb for what to do.

Reply:
Done.